otterdog/adoptium.jsonnet

```diff
@@ -580,6 +580,7 @@ orgs.newOrg('adoptium') {
     },
     newMirrorRepo('jdk21u') {},
     newMirrorRepo('jdk22') {},
```

Comment:
do we want to archive this one too?

Reply:
Yeah I was asking that question in slack too as it's not yet in the docs to do so :-) Worst case I was going to do it as a separate PR but happy to updte this one for that now.